mini_githubcic/mini_githubcic/views.py

```diff
@@ -262,6 +257,19 @@ def form_valid(self, form):
 
         return super().form_valid(form)
 
+    def get_context_data(self, *args, **kwargs):
+        context = super(IssueUpdateView, self).get_context_data(**kwargs)
+        issue = Issue.objects.filte(self.request.resolver_match.kwargs['pk']).first()
```

Comment:
causes error, change to:
`        issue = Issue.objects.filter(id=int(self.request.resolver_match.kwargs['pk'])).first()
`